### PR TITLE
fix path in burden test

### PR DESCRIPTION
**Pull request by dianacornejo**
fix some path issues burden test

### CODE REVIEW

File: GWAS/LMM.ipynb [hunk 1]
```diff
@@ -2226,7 +2226,7 @@
     "        topgene<-data[data$P<=${plim},]    \n",
     "    }  \n",
     "    if (${1 if k !=\"\" else 0} ){\n",
-    "        topgene<-data[order(data$P),][1:${k},]    \n",
+    "        topgene<-data[order(data$P),][1:as.double(${k}),]    \n",
```
Comment (gaow) [inline GWAS/LMM.ipynb:2229]: It does not make sense to use double for an index. Why would you do that? What's wrong with using 1:${k}?

Comment (dianacornejo) [reply GWAS/LMM.ipynb:2229]: @gaow it was giving this error and @UxxUnet suggested to do this and it worked after changing it
```
Error: unexpected ',' in:
"if (0 ){
    topgene<-data[order(data$P),][1:,"
Execution halted
Error: unexpected ',' in:
"if (0 ){
    topgene<-data[order(data$P),][1:,"
Execution halted
Error: unexpected ',' in:
"if (0 ){
    topgene<-data[order(data$P),][1:,"
Execution halted
Error: unexpected ',' in:
"if (0 ){
    topgene<-data[order(data$P),][1:,"
Execution halted
```

**Comment by gaow** [comment]
I see. Let me merge and fix it properly

**Comment by gaow** [comment]
@dianacornejo @UxxUnet see my patch [here](https://github.com/cumc/bioworkflows/commit/9fef882f8b9026bd1dd470167f6e835991b5bfcb)

**Pull request merged**